scripts/patched-jest-resolver.js

```diff
@@ -1,7 +1,9 @@
 'use strict';
 Object.defineProperty(exports, '__esModule', { value: true });
+const { fs } = require('memfs');
```

Comment:
Can this be removed? It doesn't appear to be used anywhere

Reply:
Indeed, must have been some automatic import i overlooked
I added a fixup
fc0a44275 - (HEAD -> make-workspace-test-run, origin/make-workspace-test-run) fixup! fix(testing): make sure NX_WORKSPACE_ROOT_PATH exists before running tests (44 seconds ago) <Jon Hammerskov>